optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -156,6 +174,35 @@ public void setScoreDrlFileList(List<File> scoreDrlFileList) {
         this.scoreDrlFileList = scoreDrlFileList;
     }
 
+    public Boolean getCompileDroolsAlphaNetwork() {
+        return compileDroolsAlphaNetwork;
+    }
+
+    public void setCompileDroolsAlphaNetwork(Boolean compileDroolsAlphaNetwork) {
+        this.compileDroolsAlphaNetwork = compileDroolsAlphaNetwork;
+    }
+
+    public boolean isDroolsAlphaNetworkCompilerEnabled() {
+        if (!isUsingDrools()) {
+            LOGGER.trace("Drools Alpha Network compiler is disabled when not using Drools.");
```

Comment:
An is***() method should not be doing logs like that. If it's used twice, we log twice.
A solvingStarted can - if desired - if the alpha network is enabled. But probably on a line along with other score director information (for example number of constraints) to avoid a verbose log.

See how much info the solvingSTarted line has